### PR TITLE
Updated info.json

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
- Removed `exported_from` and `exported_by` keys
- Marked SP as `uncertified`
- Removed connector versions from content entry
- Removed utilities connector as this is system connector
- Added connector names in solution pack description

### CODE REVIEW

File: connectors/data.json [hunk 1]
```diff
@@ -27,87 +26,28 @@
...
-        "install_mode": "rpm",
-        "installer_path": "cyops-connector-carbonblack-response-2.0.1"
+        "install_mode": "rpm"
     },
     {
         "name": "ipstack",
         "label": "IPStack",
         "category": "Information",
         "description": "IPStack provides geolocation facility for IP Address or Domain.",
         "publisher": null,
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:36]: Why we are adding connector publisher here?

Comment (cs-suyog-jinturkar) [reply connectors/data.json:36]: and category?

Comment (cs-suyog-jinturkar) [reply connectors/data.json:36]: and description?

Comment (cs-suyog-jinturkar) [reply connectors/data.json:36]: IMO, its not needed. If anything changes in connector will impact this json as well. Please check with Aakash and Pooja.

File: connectors/data.json [hunk 2]
```diff
@@ -27,87 +26,28 @@
...
-        "version": "1.0.0",
         "operation_roles": {
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:37]: roles or actions?

File: connectors/data.json [hunk 3]
```diff
@@ -5,7 +5,6 @@
         "category": "Endpoint Protection",
         "description": "CarbonBlack Response is purpose-built for enterprise SOC and IR teams.This connector facilitates automated operation related to endpoint protection like isolate endpoint, unisolate endpoint, hunt file, terminate process etc. with CarbonBlack Response server.",
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:6]: connector details are missing in description

File: connectors/data.json [hunk 4]
```diff
@@ -27,87 +26,28 @@
...
-        "install_mode": "rpm",
-        "installer_path": "cyops-connector-cyops_utilities-3.2.1"
+        "install_mode": "rpm"
     },
     {
         "name": "virustotal",
         "label": "VirusTotal",
         "category": "Sandbox",
         "description": "VirusTotal provides a service that analyzes suspicious files and URLs and facilitates the quick detection of viruses, worms, trojans, and all kinds of malware. This connector facilitates automated operations such as scanning and analyzing suspicious files and URLs and retrieving reports from VirusTotal for files, IP addresses, and domains.",
         "publisher": null,
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:50]: Why publisher is empty here? VT is certified connector from us

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comment

**Comment by cs-rohit-khune** [comment]
@cs-suyog-jinturkar Added connector names in description

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**